src/features/defi/providers/fox-farming/components/FoxFarmingManager/Deposit/components/Confirm.tsx

```diff
@@ -132,6 +165,12 @@ export const Confirm: React.FC<StepComponentProps & { accountId: AccountId | und
     walletState.wallet,
   ])
 
+  useEffect(() => {
+    if (!hasEnoughBalanceForGas) {
```

Comment:
Nitpick but that would avoid the opt chain on the following line
```suggestion
    if (!(mixPanel && hasEnoughBalanceForGas)) {
```